cmd/qat_plugin/dpdkdrv/dpdkdrv.go

```diff
@@ -426,6 +455,14 @@ func getDeviceCapabilities(device string) (string, error) {
 		return "sym", nil
 	case "asym":
 		return "asym", nil
+	case "asym;dc":
+		return "asym-dc", nil
+	case "dc;asym":
+		return "asym-dc", nil
+	case "sym;dc":
+		return "sym-dc", nil
+	case "dc;sym":
+		return "sym-dc", nil
```

Comment:
The naming of the new resources looks reasonable. The question is do we harmonize `asym-sym` with the new naming or keep `cy`. I believe the latter is better to not cause any breakages ...

We have documentation for these in the top-level README and qat_plugin/README. Can you make the necessary updates to those as well (including the updated permalink to the sysfs abi).

Reply:
Will do.